models/commons/src/reference/presets/empty.ts

```diff
@@ -0,0 +1,6 @@
+import Reference from '../builder';
+import type { TReferenceBuilder } from '../types';
+
+const empty = (): TReferenceBuilder => Reference().id(undefined).key(undefined);
```

Comment:
Should this be just `id` now?
```suggestion
const empty = (): TReferenceBuilder => Reference().id(undefined);
```